gsw/gibbs/library.py

```diff
@@ -692,6 +697,56 @@ def enthalpy_SSO_0_CT25(p):
     return np.ma.array(enthalpy_SSO_0_CT25, mask=mask, copy=False)
 
 
+def enthalpy_SSO_0(p):
+    """
+    Enthalpy at SSO and CT(T=0) C (75-term equation)
+
+    This function calculates enthalpy at the Standard Ocean Salinty, SSO,
+    and at a Conservative Temperature of zero degrees C, as a function of
+    pressure, p, in dbar, using a streamlined version of the 75-term
+    computationally-efficient expression for specific volume, that is, a
+    streamlined version of the code "enthalpy(SA,CT,p)".
+
+    Parameters
+    ----------
+    p : array_like
+        pressure [dbar]
+
+    Returns
+    -------
+    enthalpy_SSO_0 : array_like
+                     enthalpy at (SSO, CT=0, p)
+                     [J kg :sup:`-1`]
+
+    Examples
+    --------
+    >>> import gsw
+    >>> p = np.array([10, 50, 125, 250, 600, 1000])
+    >>> gsw.library.enthalpy_SSO_0(p)
+    array([   97.26388583,   486.27439853,  1215.47517122,  2430.24907325,
+            5827.90879421,  9704.32030926])
+
+    Version
```

Comment:
I would not try to make both version live together in `master`. We have a release for the old one. If this is the path you guys want to go I'd say remove all the 48 terms parts and start coding the 75 here.

We can create a branch for the 48 term if too.